[view-transitions] Basic WebInspector support

nt1m opened the pull request:
#### ba0246d2761346e87d07fe9a436fee13818559e0
<pre>
[view-transitions] Basic WebInspector support
<a href="https://bugs.webkit.org/show_bug.cgi?id=265339">https://bugs.webkit.org/show_bug.cgi?id=265339</a>
<a href="https://rdar.apple.com/118793239">rdar://118793239</a>

Reviewed by Devin Rousso.

Add missing boilerplate for developer debugging purposes.

* Source/WebCore/animation/WebAnimationUtilities.cpp:
(WebCore::pseudoIdAsString):
* Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
(WI.CSSManager.displayNameForPseudoId):

Canonical link: <a href="https://commits.webkit.org/271116@main">https://commits.webkit.org/271116@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ae01cff84645d39e629246ba174ca23c54d0b78e

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/27400 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/6040 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/28649 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/29627 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/25081 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/27869 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/7952 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/3438 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/24887 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/27663 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/4838 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/23525 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/4213 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/4362 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/24521 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/30263 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/25037 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/24943 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/30503 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/4381 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/2525 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/28461 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/5848 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/4838 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/3537 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/4767 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
```diff
@@ -256,6 +256,16 @@ WI.CSSManager = class CSSManager extends WI.Object
             return WI.unlocalizedString("::scrollbar-corner");
```
@dcrousso @patrickangle Fwiw, I noticed these show the wrong strings. The scrollbar stuff should all be prefixed with -webkit-, I didn't touch it here, since there's likely more stuff to change.

dcrousso replied:
interesting.  yeah probably should change it.  feel free to do a followup fix if you'd like :)

nt1m replied:
Not sure to which extent I should change things without breaking compat nor how to test, so I'll leave it to someone else

dcrousso commented on Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
```diff
@@ -256,6 +256,16 @@ WI.CSSManager = class CSSManager extends WI.Object
+            return WI.unlocalizedString("::view-transition-group");
+        case CSSManager.PseudoSelectorNames.ViewTransitionImagePair:
+            return WI.unlocalizedString("::view-transition-image-pair");
+        case CSSManager.PseudoSelectorNames.ViewTransitionOld:
```
please alphabetize

dcrousso commented on Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
```diff
@@ -880,6 +890,11 @@ WI.CSSManager.PseudoSelectorNames = {
+    ViewTransition: "view-transition",
+    ViewTransitionGroup: "view-transition-group",
+    ViewTransitionImagePair: "view-transition-image-pair",
+    ViewTransitionOld: "view-transition-old",
```
please alphabetize

nt1m committed:
[view-transitions] Basic WebInspector support
https://bugs.webkit.org/show_bug.cgi?id=265339
rdar://118793239

Reviewed by Devin Rousso.

Add missing boilerplate for developer debugging purposes.

* Source/WebCore/animation/WebAnimationUtilities.cpp:
(WebCore::pseudoIdAsString):
* Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js:
(WI.CSSManager.displayNameForPseudoId):

Canonical link: https://commits.webkit.org/271116@main
